### PR TITLE
[docs] Fix broken description in both of English and Japanese

### PR DESCRIPTION
**Pull request by KengoTODA**
refs #360. Staging site is [here](http://spotbugs-in-kengo-toda.readthedocs.io/ja/read-the-docs/gradle.html).

cc: @orihalcon128

### CODE REVIEW

File: docs/locale/ja/LC_MESSAGES/gradle.po [hunk 1]
```diff
@@ -76,14 +88,4 @@ msgid ""
...
-#~ " `spotbugsMain and `spotbugsTest."
 #~ msgstr ""
-
-#~ msgid ""
-#~ "SpotBugs Gradle Plugin adds task "
-#~ "dependency from `check` to these these"
-#~ " tasks, so you can simly run "
-#~ "``./gradlew check`` to run SpotBugs."
-#~ msgstr ""
-
```
Comment (KengoTODA) [inline docs/locale/ja/LC_MESSAGES/gradle.po]: This file contains many meaningless diff, but they can be ignored.

@orihalcon128 I guess they comes from the difference between my Babel version and yours. Can I ask how you generate .po files? Not by `docs/build.sh`?

Comment (orihalcon128) [reply docs/locale/ja/LC_MESSAGES/gradle.po]: I am translating po files using OmegaT.

1. make gettext
2. sphinx-intl update -p .build/locale -l ja
3. Copy po files to source folder of OmegaT project.
4. Open the po file with OmegaT and translate it.
5. Generate the translated files.
6. Copy po files in target folder of OmegaT project to docs/locale/ja/LC_MESSAGES folder.

For more information on this see http://omegat.sourceforge.net/manual-standard/en/chapter.instant.start.guide.html

Comment (KengoTODA) [reply docs/locale/ja/LC_MESSAGES/gradle.po]: Got it, thanks, it should have no problem.

**Comment by KengoTODA** [comment]
I proposed #368 to update `gradle.est` more, and it includes changes in this PR. So I will close this PR.

**Pull request closed without merging**